packages/fast-components-styles-msft/src/auto-suggest/index.ts

```diff
@@ -1,18 +1,17 @@
-import designSystemDefaults, {
-    DesignSystem,
-    withDesignSystemDefaults,
-} from "../design-system";
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
 import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
 import { AutoSuggestClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
-import { elevation, ElevationMultiplier } from "../utilities/elevation";
 import { toPx } from "@microsoft/fast-jss-utilities";
+import { applyFloatingCornerRadius } from "../utilities/border";
 import { neutralFillStealthRest } from "../utilities/color";
-import { curry } from "lodash-es";
+import { heightNumber } from "../utilities/density";
+import { elevation, ElevationMultiplier } from "../utilities/elevation";
 
 const styles: ComponentStyles<AutoSuggestClassNameContract, DesignSystem> = (
     config: DesignSystem
 ): ComponentStyleSheet<AutoSuggestClassNameContract, DesignSystem> => {
     const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const visibleOptionCount: number = 10;
```

Comment:
Can we give this a better name, perhaps `visibleLines` if that's what these are

Reply:
The child object is "auto suggest option". Maybe visibleChildCount?

Reply:
I think that works